eslint/babel-eslint-plugin/README.md

```diff
@@ -28,14 +26,14 @@ original ones as well!).
 ```json
 {
   "rules": {
-    "babel/new-cap": 1,
-    "babel/camelcase": 1,
-    "babel/no-invalid-this": 1,
-    "babel/object-curly-spacing": 1,
-    "babel/quotes": 1,
-    "babel/semi": 1,
-    "babel/no-unused-expressions": 1,
-    "babel/valid-typeof": 1
+    "babel/new-cap": "error",
```

Comment:
The rule name should be prefixed by the plugin name: i.e. `@babel/eslint-plugin/new-cap`. Could you check if `@babel/new-cap` works?

Reply:
Sure - my intention isn't to update everything here, but to just get a baseline so I can start working. I was imaging that a lot of this README is going to be changed anyway, since I'm hoping to remove the deprecated rules and rethink how we keep this up-to-date with ESLint core rules.